content/docs/buildpack-author-guide/create-buildpack/building-blocks-cnb.md

```diff
@@ -9,16 +9,32 @@ Now we will set up the buildpack scaffolding.
 
 Let's create the directory where your buildpack will live:
 
+## Using the Pack CLI
+
+The `buildpack new <id>` commmand will create a directory named for the buildpack ID.
+
+Example:
 <!-- test:exec -->
 ```bash
-mkdir ruby-buildpack
+pack buildpack new examples/ruby \
+    --api 0.5 \
+    --path ruby-buildpack \
+    --version 0.0.1
```

Comment:
Do we need to add 

`--stacks io.buildpacks.samples.stacks.bionic` here?

Reply:
Yeah you are right, i added Thx. 💯